azure/activity_logs_monitoring/index.js

```diff
@@ -524,11 +543,18 @@ class EventhubLogHandler {
         return id;
     }
 
-    extractMetadataFromResource(record) {
-        var metadata = { tags: [], source: '' };
+    extractMetadataFromLog(record) {
+        if (shouldParseDefenderForCloudLogs() && this.isDefenderForCloudLog(record)) {
+            return this.extractMetadataFromDefenderLog(record);
+        }
+        return this.extractMetadataFromStandardLog(record);
+    }
+
+    extractMetadataFromStandardLog(record) {
+        var metadata = { tags: [], source: '', service: '' };
         var resourceId = this.getResourceId(record);
         if (resourceId === null || resourceId === '') {
-            return metadata;
+            return [metadata, record];
```

Comment:
Is there a reason to return the record as well here? it looks like the only usage in `extractMetadataFromLog`, so you could just `return [this.extractMetadataFromStandardLog(record), record]` there, since you don't modify it in the function it looks like